packages/core/networks/1.json

```diff
@@ -123,19 +123,6 @@
     "contractName": "SuccessTokenLongShortPairFinancialProductLibrary",
     "address": "0x67DE29D1A34FF2ea2b8C390B326456F4CBBE628F"
   },
-  {
```

Comment:
these contracts are deprecated, so we should remove them from the networks page.